### PR TITLE
GSoC: Added beta sobolev module

### PR DESCRIPTION
**Pull request by aoifeboyle**
Work in progress do not merge (for now).
Just doing this PR for the GSoC application. Expanded the stub of the beta sobolev plasma property module. 

TODO
 - [x] initial documentation
 - [ ] ion population needs some changes

### CODE REVIEW

File: tardis/plasma/properties/level_population.py [hunk 1]
```diff
@@ -128,4 +128,52 @@ def calculate(self):
             x[0] = 1.0
             for i in xrange(len(self.t_rads)):
                 relative_level_populations = np.linalg.solve(rates_matrix[:, :, i], x)
-                self.level_populations[i].ix[species] = relative_level_populations * self.ion_populations[i].ix[species]
\ No newline at end of file
+                self.level_populations[i].ix[species] = relative_level_populations * self.ion_populations[i].ix[species]
+
+#In Progress: Approximation to account for ionisation of helium by fast electron collisions. Boyle et al., 2015
+class HeliumLevelPopulationsNLTE(ProcessingPlasmaProperty):
+
```
Comment (wkerzendorf) [inline tardis/plasma/properties/level_population.py]: hi you can add docs here.

**Comment by aoifeboyle** [comment]
@wkerzendorf I added a new file to the 'docs' folder about it. How's that? By the way, in answer to your question, I like the new plasma layout :) It seems a lot cleaner than the old way.

File: docs/physics/plasma/helium_nlte_approx.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Helium Non-Thermal Treatment Approximation (ab-NLTE)
+------------------------------------------
```
Comment (wkerzendorf) [inline docs/physics/plasma/helium_nlte_approx.rst]: That's even more than I was thinking about - Very nice!!

File: tardis/plasma/properties/level_population.py (hunk 1 above)
Comment (wkerzendorf) [inline tardis/plasma/properties/level_population.py]: you can give a short documentation for a class using this

```python
class XYZ:
    """this describes what xyz does 
    Parameters
    ----------------

    a: float
        this is a really important parameter ....
```

This is more a general comment.

**Comment by wkerzendorf** [comment]
That's what we are going for. There's still quite a bit of work to do but - I think it's going in the right direction.

**Comment by aoifeboyle** [comment]
@wkerzendorf I updated this with the changes you suggested. I hope I understood what you meant correctly.

File: tardis/plasma/properties/level_population.py [hunk 3]
```diff
@@ -128,4 +128,54 @@ def calculate(self):
             x[0] = 1.0
             for i in xrange(len(self.t_rads)):
                 relative_level_populations = np.linalg.solve(rates_matrix[:, :, i], x)
-                self.level_populations[i].ix[species] = relative_level_populations * self.ion_populations[i].ix[species]
\ No newline at end of file
+                self.level_populations[i].ix[species] = relative_level_populations * self.ion_populations[i].ix[species]
+
+'''
+#In Progress: Approximation to account for ionisation of helium by fast electron collisions. Boyle et al., 2015
```
Comment (wkerzendorf) [inline tardis/plasma/properties/level_population.py]: Hi Can you take this out and put it in a different PR?

**Comment by aoifeboyle** [comment]
Closed this PR because I'm re-organising my Tardis branches and I'm going to change a few things. I'll create new, more specific PRs soon.
